src/lib/styles/global/fonts.module.scss

```diff
@@ -1,5 +1,5 @@
 /// URL to load fonts
-$fonts-path: 'https://d3kott46cqreik.cloudfront.net/assets/fonts';
+$fonts-path: 'https://d1muvgoqe3g8vw.cloudfront.net/assets/fonts';
```

Comment:
@kbardi should we set up an `$ASSETS_URL` variable like we do in the website so that it's consistent between our projects? and so this then should be something like `$fonts-path: $assets_url+'/fonts';` ... ?

Reply:
Further on this... Is `https://d1muvgoqe3g8vw.cloudfront.net/assets/fonts` right? where is this bucket? There is no `assets` folder in the root of the `aimementoring` s3 bucket. // On the website, we're using: `$cdnRoot: "https://d1muvgoqe3g8vw.cloudfront.net/website/` ... Where I'm able to navigate to the font file in the `website/assets` folder (but it's not the right one) ... 
Shouldn't the luna font be sitting in a universal bucket (not in website/blueprint/etc)...

https://www.loom.com/share/e01b8343e8354893a35a6f7ee737c7c4

Reply:
UPDATE: I think we've made a mistake here... WATCH THIS NEXT:

https://www.loom.com/share/e26ff3766b084de98fea5bb9336a47a9